Makefile

```diff
@@ -25,6 +25,10 @@ test:
 	@echo "Running testsuite"
 	go test ./...
 
+build-assets:
```

Comment:
Willst Du evtl hier nochmal was am naming machen? Wir haben jetzt build, build-assets und assets. Wie wärs hier einfach mit ui?

Reply:
Ja find ich gut. Ich benenne den Task in `ui` um und löse die Konflikte.